common/lib/xmodule/xmodule/course_module.py

```diff
@@ -934,6 +934,15 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    pay_now_link = String(
+        display_name=_(" Pay-Now Link "),
+        help=_(
+            "This replaces the enrol button on the course in order to redirect to an external payment page "
```

Comment:
Please remove the space at the end of the line and finalize it with a period. @andrey-canon